### ISSUE TITLE
Moving docs to gh-pages

### ISSUE THREAD

**Issue by asclines** [TTUSDC/cpceed-webapp#12]
So Travis offers a pretty simple way to deploy to a GitHub branch. [link](https://docs.travis-ci.com/user/deployment/pages/)

It would look cleaner to have our documentation separated from the main project files. I know I personally wanted the documentation to follow the project so that when one was looking at prior versions of the project, they would have the documentation to go with. However, that could still be accomplished in GitHub versioning and tags by bundling the documentation into the release then. 

I propose we add something similar to this into `.travis.yml`:
```yml
deploy:
  provider: pages
  skip_cleanup: true
  github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
  local_dir: docs/
  on:
    branch: master
```

**Comment by NilsG-S** [comment]
Sounds good to me!

**Comment by asclines** [comment]
Closed in #14
